src/Pate/Verification.hs

```diff
@@ -509,9 +509,9 @@ checkEquivalence triple = startTimer $ withSym $ \sym -> do
 withSimBundle ::
   (HasCallStack, PEM.ExprMappable sym f) =>
   PPa.BlockPair arch ->
-  (SimBundle sym arch -> EquivM sym arch f) ->
-  EquivM sym arch (SimSpec sym arch f)
-withSimBundle pPair f = withEmptyAssumptionFrame $ withSym $ \sym -> do
+  (SimBundle sym arch -> EquivM sym arch (f, prf)) ->
+  EquivM sym arch (SimSpec sym arch f, prf)
```

Comment:
How should I interpret this return value? What is the concept I should have in mind when calling `withSimBundle`?

Reply:
Added some documentation